api/models/config_model.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+// Config -
+type Config struct {
+	LedgerStorage *LedgerStorage `json:"storage"`
```

Comment:
I think it could make sense to move the Config struct and its children to the root `config` directory. What do you think?

Reply:
Config struct is now accessible from config.go